### ISSUE TITLE
[Unit Testing] Need to test all available functions in latest version

### ISSUE THREAD

**Issue by kashifkhan0771** [kashifkhan0771/utils#41]
We should test out all available functions in the latest version as of now `v0.7.0`. Testing can be done anywhere. You can import the library and test each function properly. Any sort of evidence for successful testing will be accepted. If you find any bugs/issues during testing, feel free to open a new issue.

**Comment by JoseScap** [comment]
@kashifkhan0771 could you assign to me this feature?

**Comment by kashifkhan0771** [comment]
Sure @JoseScap - Thanks you!

**Comment by kashifkhan0771** [comment]
I would suggest adding `REPORT.Json` file after testing. This way your contribution will be noted as well. You can follow following pattern for Report file.
```json
[
  {
    "version": "v0.7.0",
    "performed_on": "dd-mm-yyyy",
    "performed_by": "github_username",
    "no_of_issues_found": 0,
    "tested_features": [
      "random",
      "maps"
    ],
    "issue": "link to issue"
  }
]
```
Make sure to add any sort of evidence for testing in the issue comments.

**Comment by kashifkhan0771** [comment]
Once this testing is complete we will draft our first stable release `v1.0.0`

**Comment by kashifkhan0771** [comment]
Any update on this @JoseScap ?

**Comment by kashifkhan0771** [comment]
I guess @JoseScap got busy with something, can anyone else take this? We need to release our first stable version after this.

**Comment by kashifkhan0771** [comment]
I guess, I have to do it myself :)

**Comment by kashifkhan0771** [comment]
Closing this issue and will release the `v1.0.0`. IMO we have unit tests in place so code should be fine. If we found any issues, the bug can be reported by anyone.
